src/components/Inputs/DocumentInput/DocumentInput.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+  <div class="pb-document-input-container">
+    <input
+      v-model="documentInput"
+      class="pb"
+      style="width: 100%"
+      :class="validationClass"
+      :style="documentInputStyle"
+      :placeholder="setInputPlaceholder"
+      :maxlength="getMaxLength(inputType)"
+      @blur="validateDocument"
+      @focus="updateValidationFiled"
+    >
+    <p
+      v-if="state.message"
+      class="pb error-text"
+    >
+      {{ state.message }}
+    </p>
+  </div>
+</template>
+
+<script>
+import { validateColor } from '@pb/utils/validator';
+import { isCpf, isCnpj } from 'adapcon-utils-js';
+
+export default {
+  name: 'PbDocumentInput',
+
+  props: {
+    required: { type: Boolean, default: false },
+    value: { type: String, default: '' },
+    background: { type: String, default: 'transparent' },
+    color: { type: String, default: 'gray-20', validator: validateColor },
+    inputType: { type: Array, default: () => ['cnpj'] },
```

Comment:
Use a validator to check the inputType values (example src\components\Inputs\TextArea\TextArea.vue -> validateColor)